packages/storybook-builder-vite/source-loader-plugin.js

```diff
@@ -1,24 +1,18 @@
-const inject =
-    require('@storybook/source-loader/dist/cjs/abstract-syntax-tree/inject-decorator').default;
+const transform =
```

Comment:
Hmmmm, I'm not sure I love using the same name as the `transform` function below.  Would you mind naming it something else, like maybe `sourceLoaderTransform` or something?